.gitignore

```diff
@@ -17,3 +17,4 @@ lib/types/*
 !lib/esm/package.json
 !lib/cjs/package.json
 convert-units-*.tgz
+yarn.lock
```

Comment:
The project doesn't use yarn as the package manager, so including this in the gitignore is redundant.

Reply:
Done. Please see latest commit.